pkg/statdb/statdb.go

```diff
@@ -52,4 +52,12 @@ type NodeStats struct {
 	UptimeRatio        float64
 	UptimeSuccessCount int64
 	UptimeCount        int64
+	Wallet             string
+	Email              string
+}
+
+// Meta is a struct for organizing the meta data for a node operator
+type Meta struct {
```

Comment:
We have the `pb.NodeOperator` type with the same two fields. Is it really necessary to define this new type here?

Reply:
I didn't realize pb.NodeOperator had those fields on them. I can try to replace with pb.NodeOperator.